### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.7, 3.8 and 3.9.
* Drop support for Python 3.4.

### CODE REVIEW

**Comment by icemac** [comment]
Drop in coverage seems to come from the switch to branch coverage.

File: src/zope/app/rotterdam/__init__.py [hunk 1]
```diff
@@ -17,18 +17,21 @@
...
 
+
 class Rotterdam(rotterdam, IDefaultBrowserLayer):
     """The ``Rotterdam`` skin.
 
     It is available via ``++skin++Rotterdam``.
     """
 
+
 # BBB 2006/02/18, to be removed after 12 months
```
Comment (sallner) [inline src/zope/app/rotterdam/__init__.py]: It is actually a lot later than 12 months...

**Review by sallner — APPROVED** [review]
Looks good to me, `buildout.cfg` can be removed.

File: src/zope/app/rotterdam/__init__.py (hunk 1 above)
Comment (icemac) [reply src/zope/app/rotterdam/__init__.py]: Removed in https://github.com/zopefoundation/zope.app.rotterdam/pull/3/commits/e71ae576bf3d59692a962540efa72ab6f1572d4a.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**